packages/flag-evaluation/src/index.ts

```diff
@@ -342,6 +344,16 @@ export function evaluate(
         ? fieldValueDate > daysAgo.getTime()
         : fieldValueDate < daysAgo.getTime();
     }
+    case "DATE_AFTER": {
```

Comment:
you can probably combine these two like in the case of `AFTER` and `BEFORE`. And only change the operator

Reply:
Good point